src/EnergyPlus/General.cc

```diff
@@ -91,6 +91,70 @@ using DataHVACGlobals::Bisection;
 // MODULE PARAMETER DEFINITIONS
 static constexpr std::string_view BlankString;
 
+enum class ReportType
```

Comment:
Good to define these in the `.cc` file if they are only used within this module. Type scope reduction is also good. All scope reduction is good.